docs/getting-started-guides/mesos.md

```diff
@@ -191,6 +193,95 @@ Verify that the pod task is running in the Mesos web GUI. Click on the
 Kubernetes framework. The next screen should show the running Mesos task that
 started the Kubernetes pod.
 
+## Launching kube-dns
+
+Kube-dns is an addon for Kubernetes which adds service discovery to the cluster. For a detailed explanation see [DNS in Kubernetes][4].
```

Comment:
service discovery -> DNS-based service discovery (the env var based service discovery mechanism, while primitive, is available without DNS)